Docs - add mfa instructions (#904)
CloudFire-LLC/cloudfire: docs/docs/authenticate/multi-factor.md

---
title: Multi-Factor Authentication
sidebar_position: 2
---

Multi-factor authentication (MFA) can be added directly through Firezone
or by adding an additional factor directly through your identity provider.

## MFA with Firezone

Firezone currently supports using a time-based one time password

> (TOTP) as an additional factor.

Reviewer: Maybe a reminder this is for local email/password only.

```suggestion
(TOTP) as an additional factor. This is supported with the local authentication method only; for SSO authentication we recommend enabling your provider's MFA functionality [as described below](#identity-provider).
```

> Admins can visit `/settings/account` in the admin portal to generate a

Reviewer: ```suggestion
Admins can visit `/settings/account/register_mfa` in the admin portal to generate a
```

QR code to be scanned by your authenticator app.

> Unprivileged users can visit the **My Account** page after logging into

Reviewer: ```suggestion
Unprivileged users can visit `/user_account/register_mfa` after logging into
```

the user portal.

> ## Identity Provider

Reviewer: ```suggestion
## MFA with Identity Provider
```

> MFA can be configured through your identity provider if you have an existing
> integration for SSO. Consult your provider's documentation to enforce an
> additional factor. We have included links to a few common providers below:

Reviewer: ```suggestion
Most identity providers support additional authentication factors in addition to email/password. Consult your provider's documentation to enforce an
additional factor. We have included links to a few common providers below:
```
